src/services/members/index.ts

```diff
@@ -0,0 +1,17 @@
+import { MEMBERS_API } from "@/const";
+
+export interface MemberData {
+  name: string;
+  githubUrl: string;
+  bio: string;
+  imageUrl: string;
+  interests: string[];
+}
+
+export const getMemberInfo = async () => {
+  const res = await fetch(MEMBERS_API);
+  const responseJSON = await res.json();
+  const memberData = responseJSON.data as MemberData[];
+
+  return memberData;
+};
```

Comment:
[let's try to avoid type assertion when we can](https://mainawycliffe.dev/blog/type-assertions-in-typescript/)
```suggestion
export const getMemberInfo = async (): Promise<MemberData[]> => {
  const res = await fetch(MEMBERS_API);
  const responseJSON = await res.json();
  const memberData = responseJSON.data;

  return memberData;
};
```

Reply:
Noted!